Makefile

```diff
@@ -84,8 +85,14 @@ else
 TESTCMD = -run $(RUNTEST)
 endif
 
+test_skip_kube: SKIP_KUBE=1
+test_skip_kube: test
```

Comment:
These targets are unnecessary so they should be removed, but in the future, stuff like this should be added under `.PHONY`.